pipelines/jobs/configurations/jdk20.groovy

```diff
@@ -0,0 +1,59 @@
+targetConfigurations = [
+        "x64Mac"      : [
+                "temurin"
+        ],
+        "x64Linux"    : [
+                "temurin"
+        ],
+        "x64AlpineLinux" : [
+                "temurin"
+        ],
+        "x64Windows"  : [
+                "temurin"
+        ],
+        "x32Windows"  : [
+                "temurin"
+        ],
+        "ppc64Aix"    : [
+                "temurin"
+        ],
+        "ppc64leLinux": [
+                "temurin"
+        ],
+        "s390xLinux"  : [
+                "temurin"
+        ],
+        "aarch64Linux": [
+                "hotspot",
```

Comment:
what's the reason for the `hotspot` build?

Reply:
it's from jdk-19, added here: https://github.com/adoptium/ci-jenkins-pipelines/pull/251
@sxa presume this is actually good here since this is jdk(head), but should we remove it from jdk-19 ?

Reply:
Yes - I'd say so. Not sure if we can add inline comments into these files but might be good to put a note saying "I'f you've moved this out of JDK head you might want to remove this one"